opentelemetry-jaeger/src/lib.rs

```diff
@@ -622,7 +622,7 @@ fn build_span_tags(
     }
 
     // Ensure error status is set
-    if status_code != StatusCode::OK && !user_overrides.error {
+    if status_code != StatusCode::Ok && !user_overrides.error {
```

Comment:
This should probably instead look for error now

Reply:
done.